audiobook/main.py

```diff
@@ -70,7 +66,7 @@ def create_json_book(self, input_book_path, password=None, extraction_engine=Non
         )
 
         if os.path.exists(os.path.join(BOOK_DIR, json_filename)):
-            metadata = {}
+            metadata = {"book_name": json_filename.split(".")[0]}
```

Comment:
Missing `book_name` in `metadata` was triggering an exception when saving audio in case the `json` file already existed (with no `book_name`)